helm_deploy/values-prod.yaml

```diff
@@ -35,6 +35,7 @@ env:
   HMPPS_COOKIE_NAME: hmpps-session-prod
   REMOTE_AUTH_STRATEGY: "true"
   WEB_SESSION_TIMEOUT_IN_MINUTES: "60"
+  WHEREABOUTS_URL: "https://whereabouts.prison.service.justice.gov.uk/"
```

Comment:
this doesn't look right - you've got a trailing slash here but not on the other two.  We're trying not to have trailing slashes on our endpoints...

Reply:
see case notes example that has been done recently